src/styles/breeding.less

```diff
@@ -1,3 +1,75 @@
+/* ----------------------------------------------
+ * Generated by Animista on 2020-10-3 20:59:33
+ * Licensed under FreeBSD License.
+ * See http://animista.net/license for more info. 
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+
+/**
+ * ----------------------------------------
+ * animation wobble-hor-bottom
+ * ----------------------------------------
+ */
+@-webkit-keyframes wobble-hor-bottom {
+  0%,
+  100% {
+    -webkit-transform: translateX(0%);
+            transform: translateX(0%);
+    -webkit-transform-origin: 50% 50%;
+            transform-origin: 50% 50%;
+  }
+  15% {
+    -webkit-transform: translateX(-10px) rotate(-6deg);
+            transform: translateX(-10px) rotate(-6deg);
+  }
+  30% {
+    -webkit-transform: translateX(5px) rotate(6deg);
+            transform: translateX(5px) rotate(6deg);
+  }
+  45% {
+    -webkit-transform: translateX(-5px) rotate(-3.6deg);
+            transform: translateX(-5px) rotate(-3.6deg);
+  }
+  60% {
+    -webkit-transform: translateX(3px) rotate(2.4deg);
+            transform: translateX(3px) rotate(2.4deg);
+  }
+  75% {
+    -webkit-transform: translateX(-2px) rotate(-1.2deg);
+            transform: translateX(-2px) rotate(-1.2deg);
+  }
+}
```

Comment:
I don't believe there's any benefit to having the `@-webkit-keyframes` anymore as none of our supported browser don't support `@keyframes` as far as I am aware.

<https://caniuse.com/mdn-css_at-rules_keyframes>

Also the same thing goes for `-webkit-transform` & `-webkit-animation` used below.

<https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions#Formerly_proprietary_properties_that_are_now_standard>